molgenis-data-import/src/main/java/org/molgenis/data/export/mapper/EntityTypeMapper.java

```diff
@@ -0,0 +1,87 @@
+import static org.molgenis.data.importer.emx.EmxMetadataParser.EMX_ENTITIES_ABSTRACT;
+import static org.molgenis.data.importer.emx.EmxMetadataParser.EMX_ENTITIES_BACKEND;
+import static org.molgenis.data.importer.emx.EmxMetadataParser.EMX_ENTITIES_DESCRIPTION;
+import static org.molgenis.data.importer.emx.EmxMetadataParser.EMX_ENTITIES_EXTENDS;
+import static org.molgenis.data.importer.emx.EmxMetadataParser.EMX_ENTITIES_LABEL;
+import static org.molgenis.data.importer.emx.EmxMetadataParser.EMX_ENTITIES_NAME;
+import static org.molgenis.data.importer.emx.EmxMetadataParser.EMX_ENTITIES_PACKAGE;
+import static org.molgenis.data.importer.emx.EmxMetadataParser.EMX_ENTITIES_TAGS;
+
+import com.google.common.collect.Lists;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import org.apache.commons.lang3.StringUtils;
+import org.molgenis.data.meta.model.EntityType;
+import org.molgenis.data.meta.model.EntityTypeMetadata;
+import org.molgenis.data.meta.model.Package;
+import org.molgenis.data.meta.model.Tag;
+
+public class EntityTypeMapper {
+  public static final Map<String, String> ENTITIES_ATTRS = newLinkedHashMap();
+
+  static {
+    ENTITIES_ATTRS.put(EMX_ENTITIES_NAME, EntityTypeMetadata.ID);
+    ENTITIES_ATTRS.put(EMX_ENTITIES_PACKAGE, EntityTypeMetadata.PACKAGE);
+    ENTITIES_ATTRS.put(EMX_ENTITIES_LABEL, EntityTypeMetadata.LABEL);
+    ENTITIES_ATTRS.put(EMX_ENTITIES_DESCRIPTION, EntityTypeMetadata.DESCRIPTION);
+    ENTITIES_ATTRS.put(EMX_ENTITIES_ABSTRACT, EntityTypeMetadata.IS_ABSTRACT);
+    ENTITIES_ATTRS.put(EMX_ENTITIES_EXTENDS, EntityTypeMetadata.EXTENDS);
+    ENTITIES_ATTRS.put(EMX_ENTITIES_BACKEND, EntityTypeMetadata.BACKEND);
+    ENTITIES_ATTRS.put(EMX_ENTITIES_TAGS, EntityTypeMetadata.TAGS);
+  }
+
+  private EntityTypeMapper() {}
+
+  public static List<Object> map(EntityType entityType) {
+    List<Object> row = Lists.newArrayList();
+    for (Entry<String, String> entry : ENTITIES_ATTRS.entrySet()) {
+      switch (entry.getKey()) {
+        case EMX_ENTITIES_TAGS:
+          row.add(getTags(entityType));
+          break;
+        case EMX_ENTITIES_EXTENDS:
+          row.add(entityType.getExtends() != null ? entityType.getExtends().getId() : "");
+          break;
+        case EMX_ENTITIES_PACKAGE:
+          row.add(getPackageId(entityType));
+          break;
+        case EMX_ENTITIES_NAME:
+          row.add(getName(entityType));
+          break;
+        default:
+          Object value = entityType.get(entry.getValue());
+          row.add(value != null ? value.toString() : "");
+          break;
+      }
+    }
+    return row;
+  }
+
+  private static String getTags(EntityType entityType) {
+    return StringUtils.join(
+        StreamSupport.stream(entityType.getTags().spliterator(), false)
+            .map(Tag::getId)
+            .collect(Collectors.toList()),
```

Comment:
Static import Collectors.toList:

```suggestion
            .collect(toList());
```